.travis.yml

```diff
@@ -3,5 +3,5 @@ sudo: false
 language: go
 go:
   - 1.4.3
-  - 1.5.1
+  - 1.5.4
```

Comment:
Given that this repository is about libs, should we continue to test against `go 1.4.3` and also add `go 1.6.2` ?